language_detector/main.py

```diff
@@ -1,9 +1,25 @@
+# -​*- coding: utf-8 -*​-
+
 """This is the entry point of the program."""
 
 from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    languages = {'German': 0, 'English': 0, 'Spanish': 0}
+    
+    text_list = text.split()
+    for words in text_list:
+        for dictionaries in LANGUAGES:
+            if words in dictionaries['common_words']:
+                if dictionaries['name'] == 'Spanish':
+                    languages['Spanish'] += 1
```

Comment:
If the number of languages increase (let's say 20), you would have to add so much ``` if dictionaries['name'] == 'Spanish': ``` kind of conditions. 
Get the language name first, and then directly increase the value part of the dictionary item.